(DOCS-15459): Change Stream History Lost Trigger Errors (#99)
mongodb/docs-app-services: source/triggers/database-triggers.txt

  such as ``dropDatabase``, ``renameCollection``, or those caused by
  a network disruption
- the token required to resume the change stream is no longer in the

>   cluster :manual:`oplog </core/replica-set-oplog/>`, called a

Author: Fixed!

>   cluster :manual:`oplog </core/replica-set-oplog/>`, called a
>   ``ChangeStreamHistoryLost`` error in the App logs

Author: ```suggestion
  cluster :manual:`oplog </core/replica-set-oplog/>`. The App logs
  refer to this as a ``ChangeStreamHistoryLost`` error.

```
